jbmc/unit/java_bytecode/goto_program_generics/mutually_recursive_generics.cpp

```diff
@@ -0,0 +1,146 @@
+/*******************************************************************\
+
+ Module: Unit tests for parsing mutually generic classes
+
+ Author: Diffblue Ltd.
+
+\*******************************************************************/
+#include <iostream>
+#include <java-testing-utils/load_java_class.h>
+#include <java-testing-utils/require_goto_statements.h>
+#include <java-testing-utils/require_type.h>
+#include <testing-utils/catch.hpp>
+
+SCENARIO(
+  "Generics class with mutually recursive_generic parameters",
+  "[core][java_bytecode][goto_programs_generics]")
+{
+  const symbol_tablet &symbol_table = load_java_class(
+    "MutuallyRecursiveGenerics",
+    "./java_bytecode/goto_programs_generics",
```

Comment:
I think your CI is failing due to a typo here, it should be `goto_program_generics`, not `goto_programs_generics`